models/articles.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+function selectArticleById(article_id) {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then((results) => {
+      if (!results.rowCount)
+        return Promise.reject({ status_code: 404, msg: "Not found" });
+
+      return results.rows;
```

Comment:
sned back rows[0] so you don't have to access array in the controller (all data formating should be done in model)

Reply:
Has now been refactored.